cli/lib/exec/spawn.js

```diff
@@ -16,7 +16,21 @@ const isXlibOrLibudevRe = /^(?:Xlib|libudev)/
 const isHighSierraWarningRe = /\*\*\* WARNING/
 const isRenderWorkerRe = /\.RenderWorker-/
 
-const GARBAGE_WARNINGS = [isXlibOrLibudevRe, isHighSierraWarningRe, isRenderWorkerRe]
+// Chromium (which Electron uses) always makes several attempts to connect to the system dbus.
+// This works fine in most desktop environments, but in a docker container, there is no dbus service
+// and Chromium emits several error lines, similar to these:
+
+// [1957:0406/160550.146820:ERROR:bus.cc(392)] Failed to connect to the bus: Failed to connect to socket /var/run/dbus/system_bus_socket: No such file or directory
+// [1957:0406/160550.147994:ERROR:bus.cc(392)] Failed to connect to the bus: Address does not contain a colon
+
+// These warnings are absolutely harmless. Failure to connect to dbus means that electron won't be able to access the user's
+// credential wallet (none exists in a docker container) and won't show up in the system tray (again, none exists).
+// Failure to connect is expected and normal here, but users frequently misidentify these errors as the cause of their problems.
+
+// https://github.com/cypress-io/cypress/issues/19299
+const isDbusWarning = /Failed to connect to the bus:/
```

Comment:
Do you think this is an adequate Regex? I guess it's highly unlikely someone would have "Failed to connect to the bus:" as some other output that would be stripped. Feels a bit less specific than the other ones stripped though.

Reply:
This regex is applied pretty narrowly: to the stdout/stderr from electron itself. This doesn't filter any user-generated messages or things coming from our server. I settled on this based on the chromium source code:

```
  if (!connection_) {
    LOG(ERROR) << "Failed to connect to the bus: "
               << (dbus_error_is_set(error.get()) ? error.message() : "");
    return false;
  }
```
https://chromium.googlesource.com/chromium/src/+/524f7d22d706ccf6264dbe0ea90a5b469f2c3379/dbus/bus.cc#392

I think it's about as specific as is reasonable here - ideally we won't need to update it as we change electron versions, so the 'technical' stuff up front (`[1957:0406/160550.147994:ERROR:bus.cc(392)]`) doesn't seem like a good candidate for inclusion.